Common/src/geometry/CPhysicalGeometry.cpp

```diff
@@ -7975,6 +7975,12 @@ void CPhysicalGeometry::MatchPeriodic(CConfig        *config,
   su2double rotMatrix[3][3] = {{1.0,0.0,0.0},{0.0,1.0,0.0},{0.0,0.0,1.0}};
   su2double Theta, Phi, Psi, cosTheta, sinTheta, cosPhi, sinPhi, cosPsi, sinPsi;
   su2double rotCoord[3] = {0.0, 0.0, 0.0};
+  
+  bool pointonAxis = false;
+  
+  bool chksamePoint = false;
+  
+  su2double dist_to_Axis = 0.0;
```

Comment:
Hi @koodlyakshay,
Please follow the most common naming convention (`pointOnAxis`, `distToAxis`).
Is this going to match points to themselves if they are on the axis?

Reply:
Hi @pcarruscag,

Fixed the naming scheme. 
Also, yes the idea is to map the CV around a node on the axis on one periodic face with the corresponding CV around the same node on the other periodic face.